pkgs/tools/networking/ddclient/default.nix

```diff
@@ -2,11 +2,11 @@
 
 perlPackages.buildPerlPackage rec {
   pname = "ddclient";
-  version = "3.9.0";
+  version = "3.9.1";
 
   src = fetchurl {
     url = "mirror://sourceforge/ddclient/${pname}-${version}.tar.gz";
```

Comment:
This should fetch from GitHub instead as the tarballs on sourceforge are usually delayed.

Reply:
@Stunkymonkey can you make a new PR?

Reply:
sure. wait a few minutes